src/features/logging/instrument/index.js

```diff
@@ -12,7 +11,8 @@ export class Instrument extends InstrumentBase {
     /** emitted by wrap-logger function */
     this.ee.on('wrap-logger-end', function handleLog ([message, ...args]) {
       const { level } = this
-      bufferLog(instanceEE, message, { ...(!!args.length && { customAttributes: stringify(args) }) }, level)
+      const customAttributes = args.length ? args : [{}]
```

Comment:
`customAttributes should remain an object type rather than an array.` it is an array if supplied as the arguments from the `wrapLogger` and there's not a way around this if implemented the way you and @patrickhousley wanted it... ie stringifying the arguments.

This is an attempt to salvage some type of consistency here.